src/test/java/org/jitsi/meet/test/web/WebParticipantFactory.java

```diff
@@ -203,6 +206,9 @@ else if (participantType == ParticipantType.edge)
             logPrefs.enable(LogType.BROWSER, Level.ALL);
 
             final ChromeOptions ops = new ChromeOptions();
+            // Force chrome to use English instead of system language.
+            // At least work at version 83.0.4103.61 on win 10
+            ops.addArguments("--lang=en");
```

Comment:
I think these arguments need to be without `--`

Reply:
It should with --

Reply:
There is another solution.
```java
            Map<String, String> lops = new HashMap<>();
            lops.put("intl.accept_languages", "en,en_US");
            ops.setExperimentalOption("prefs", lops);
```
It work on my computer, too.
But `addArguments` is short, so I choose that.

Reply:
Look at the rest of the arguments like `ops.addArguments("allow-insecure-localhost");`
On the command line that is `--allow-insecure-localhost`.

Reply:
with `--` it just work. without `--` I can't promise it will still work.

Reply:
You had tested it with torture and -- ? Or you just had tested it -- command line?

Reply:
Or you are right, it does not matter https://www.selenium.dev/selenium/docs/api/java/org/openqa/selenium/chrome/ChromeOptions.html#addArguments-java.util.List-

Reply:
As you can see it the docs the examples are without --

Reply:
I have test without `--` on windows, it also works, so I removed the `--` to keep the format.